.pre-commit-config.yaml

```diff
@@ -53,12 +53,12 @@ repos:
 
   # Autoformat: markdown, yaml (but not helm templates)
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.7.1
+    rev: v3.0.0-alpha.0
```

Comment:
Can we block pre-releases from these PRs?

Reply:
Apparently, pre-commit.ci using `pre-commit autoupdate` to update this, doesn't reflect on versions - just the latest tag.

Looking at the configuration related to pre-commit.ci, [that can be declared in this file under `ci`](https://pre-commit.ci/#configuration), gives me no hope that we can avoid this nicely.